src/controllers/admin.controller.js

```diff
@@ -5,26 +5,30 @@ const universityServiceAPI = require("../services/at-university-api.service");
 
 // AT-UNIVERSITY - Admin - Index
 adminCtrl.renderIndexAdmin = (req, res) => {
-  res.render("admin/index");
+    res.render("admin/index");
 };
 
 // AT-UNIVERSITY - Admin - Render Course List
 adminCtrl.renderCourseList = async (req, res) => {
     try {
         const responseCourses = await universityServiceAPI.getAllCourses();
         console.log("---> adminCtrl.renderCourseList.getAllCourses");
-        //console.log(responseCourses.data);
         const courses = responseCourses.data;
-        res.render("admin/course/index", { courses });

```

Comment:
Why did you delete it?

Reply:
"/ index" is not being executed and with that line or without that line it continues executing without "/ courses"